### ISSUE TITLE
bots do not try to prevent obvious win

### ISSUE THREAD

**Issue by ronaldossendrijver** [ronaldossendrijver/treachery.online#45]

**Comment by ronaldossendrijver** [comment]
[savegame (48).json](https://github.com/ronaldossendrijver/treachery.online/files/14896724/savegame.48.json)

**Comment by C4ESIUM** [comment]
Yeah it's my main concern with bots, I often close my games earlier because my ally and the other alliance don't even try to contest the 4 strongolds where 1 alliance is into uncontested.
